avm/res/network/network-manager/security-admin-configuration/rule-collection/rule/main.bicep

```
@@ -29,10 +29,10 @@ param description string = ''
 param access string
 
 @sys.description('Optional. List of destination port ranges. This specifies on which ports traffic will be allowed or denied by this rule. Provide an (*) to allow traffic on any port. Port ranges are between 1-65535.')
-param destinationPortRanges array = []
+param destinationPortRanges destinationPortRangesType?
```

Why the UDT if it's anyways just a string array?

```suggestion
param destinationPortRanges string[]?
```